src/Apps/Collect/Components/Collection/Header/__tests__/DefaultHeader.test.tsx

```diff
@@ -0,0 +1,95 @@
+import { defaultCollectionHeaderArtworks } from "Apps/Collect/Components/Collection/Header/__tests__/fixtures/artworks"
+import {
+  CollectionDefaultHeader,
+  getHeaderArtworks,
+} from "Apps/Collect/Components/Collection/Header/DefaultHeader"
+import { shallow } from "enzyme"
+import React from "react"
+import renderer from "react-test-renderer"
+
+describe("artworks", () => {
+  it("returns an array of repeating artworks when original array of works is too small to fill header ", () => {
+    const artworks = defaultCollectionHeaderArtworks.hits.slice(0, 3)
+    const result = getHeaderArtworks(artworks, 1308, false)
+
+    expect(result.length).toBeGreaterThan(artworks.length)
+    expect(result).toHaveLength(6)
+  })
+
+  it("returns an array of artworks of equal length as the original array received when header can fit all works ", () => {
+    const artworks = defaultCollectionHeaderArtworks.hits
+
+    const result = getHeaderArtworks(artworks, 1308, false)
+    expect(artworks.length).toEqual(result.length)
+    expect(result).toHaveLength(10)
+  })
+})
+
+describe("default header component", () => {
+  let props
+
+  beforeEach(() => {
+    props = {
+      headerArtworks: defaultCollectionHeaderArtworks,
+      defaultHeaderImageHeight: 1000,
+    }
+  })
+
+  const getWrapper = headerProps => {
+    return shallow(<CollectionDefaultHeader {...headerProps} />)
+  }
+
+  it("renders a snapshot", () => {
```

Comment:
Is there any sense of safety we're getting out of this snapshot that we aren't getting out of our other tests? If not, I'd prefer to remove this in favor of the other more-specific tests. Snapshots of components usually result in me saying "I don't know why this is failing, so I'm just going to update it."

Reply:
Very much agree with @pepopowitz

Reply:
I find them helpful for catching unexpected UI changes personally. Before updating the snapshots I check to see why a change was triggered, and I've found its a useful tool for catching unintended changes.